Update backup-sql-server-database-azure-vms.md

supmn084 opened the pull request:
Updating notes around proxy config for the Azure Backup rules to clarify how to properly configure proxy devices for use, as most proxy devices use domain wild cards for whitelist, there are none for Azure Backup services, and you will need to rely on uploading the IP address ranges for whitelist.

supmn084 committed:
Update backup-sql-server-database-azure-vms.md

dcurwin commented on articles/backup/backup-sql-server-database-azure-vms.md:
```diff
@@ -81,7 +81,7 @@ To create a rule using PowerShell:
 **Allow access by using Azure Firewall tags**. If you're using Azure Firewall, create an application rule by using the AzureBackup [FQDN tag](https://docs.microsoft.com/azure/firewall/fqdn-tags). This allows outbound access to Azure Backup.
 
-**Deploy an HTTP proxy server to route traffic**. When you back up a SQL Server database on an Azure VM, the backup extension on the VM uses the HTTPS APIs to send management commands to Azure Backup and data to Azure Storage. The backup extension also uses Azure AD for authentication. Route the backup extension traffic for these three services through the HTTP proxy. The extensions are the only  ...
+**Deploy an HTTP proxy server to route traffic**. When you back up a SQL Server database on an Azure VM, the backup extension on the VM uses the HTTPS APIs to send management commands to Azure Backup and data to Azure Storage. The backup extension also uses Azure AD for authentication. Route the backup extension traffic for these three services through the HTTP proxy. There are no wildcard domain ...
```
"There are no wildcard domains in use with Azure Backup to add to the allow list for your proxy rules. You will need to use the public IP ranges for these services provided by Azure."

dcurwin reviewed (changes requested):
Thanks for your suggestion. I've made some small changes that I think would improve the clarity of the sentences. 

Please adjust as necessary, and I'll review your updates. Thanks!

supmn084 committed:
Update backup-sql-server-database-azure-vms.md

Updated wording for initial comments for proxy clarification. 

Updated Advantages \ Disadvantages table as it's contradicting with the ability to use storage URL's, which is not the case.

dcurwin commented on articles/backup/backup-sql-server-database-azure-vms.md:
```diff
@@ -90,7 +90,7 @@ Connectivity options include the following advantages and disadvantages:
 Use NSG service tags | Easier to manage as range changes are automatically merged <br/><br/> No additional costs <br/><br/> | Can be used with NSGs only <br/><br/> Provides access to the entire service
 Use Azure Firewall FQDN tags | Easier to manage as the required FQDNs are automatically managed | Can be used with Azure Firewall only
-Use an HTTP proxy | Granular control in the proxy over the storage URLs is allowed <br/><br/> Single point of internet access to VMs <br/><br/> Not subject to Azure IP address changes | Additional costs to run a VM with the proxy software
+Use an HTTP proxy | Single point of internet access to VMs <br/> | Additional costs to run a VM with the proxy software <br/> No published FQDN addresses to whitelist, allow rules will be subject to Azure IP address changes
```
to ~~whitelist,~~ allow rules

dcurwin reviewed (changes requested):
Change requested according to Microsoft style guides.

supmn084 committed:
Update backup-sql-server-database-azure-vms.md

Removed Whitelist from table
